packages/lexical-devtools/src/panel/components/App/index.tsx

```diff
@@ -5,24 +5,27 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
-
 import './index.css';
 
-import {EditorState} from 'lexical';
+import {DevToolsTree} from 'packages/lexical-devtools/types';
 import * as React from 'react';
 import {useEffect, useRef, useState} from 'react';
 
-import logo from '../../../images/lexical-white.png';
+import TreeView from '../TreeView';
 
 function App(): JSX.Element {
   const [count, setCount] = useState<number>(0);
-  const [, setEditorState] = useState<EditorState | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [nodeMap, setNodeMap] = useState<DevToolsTree | Record<string, never>>(
+    {},
+  );
```

Comment:
Would this just work? 

```suggestion
  const [nodeMap, setNodeMap] = useState<DevToolsTree>({});
```